packages/pricing/src/models/money-amount.ts

```diff
@@ -42,6 +42,14 @@ class MoneyAmount {
   @Property({ columnType: "numeric", nullable: true })
   max_quantity?: number | null
 
+  @Property({ columnType: "text", nullable: true })
+  price_list_id?: string
+
+  // TODO: This is added here to make the schema compatible
+  // with the core. Remove this when core is no longer running migrations
+  @Property({ columnType: "text", nullable: true })
+  region_id?: string
+
```

Comment:
question: does anything break if we don't have compatible schemas? My thinking is that it is okay that the `region_id` column is in the database table without it being available in the model here.

Reply:
I think it could be a problem if the user provides it to the module, the entity does not know about it and might throw. I would say that it still should not be part of the module and rely on the core migration table. Maybe the module should in that case strip out all unknown properties? maybe log about those properties that can't be processed by the module cause they are not supported?

Reply:
but overhaull we need to discuss how we want to handle it

Reply:
Thats actually a good point, they don't need to be in the model. I added it to the model only to auto generate the sql. I can remove these from the model.